config/dotfiles/aliases.zsh

```diff
@@ -12,9 +12,6 @@ alias cdr="cd $HOME/repos"
 alias sn='code'
 alias v='vim'
 
-alias cloudformation='aws cloudformation'
```

Comment:
I do not recall using these aliases recently.

Reply:
I verified that these aliases aren't required by other code in this repository 👍